src/project/mesa3d_desktop_intel.rs

```diff
@@ -139,6 +139,37 @@ impl Project for Mesa3DDesktopIntel {
     }
 
     fn extend_module(&self, target: &Path, module: SoongModule) -> SoongModule {
+        let mut soc_specific = SoongProp::Bool(false);
+        for lib in [
+            "libgallium_dri.so",
+            "libvulkan_intel.so",
+            "libGLESv1_CM_mesa.so.1.1.0",
+            "libGLESv2_mesa.so.2.0.0",
+            "libEGL_mesa.so.1.0.0",
+            "pps-producer",
+        ] {
+            if target.ends_with(lib) {
+                soc_specific = SoongProp::Bool(true);
+                break;
+            }
+        }
+
+        let mut relative_install_path = "";
```

Comment:
Same as `soc_specific`, we could avoid the mutable variable and set the `relative_install_path` only for modules needing it.

Reply:
Thanks, done.